### PR TITLE
Fix issue #289 - Localize Donate now and

### LINKED ISSUE #289: donate now button translation
Right now the final donate button is a bit dynamic, and I don't think we handle that very well in the translation.

"donate $3 now" needs the dollar amount injected into the string, which should have some sort of dynamic hook in the yaml file string that we can hook up to?

### PR DESCRIPTION
**Pull request by alicoding**
Closes #289

Update i18n.js and localize donate now text

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

**Comment by ScottDowne** [comment]
Looks like I landed something that conflicts with this now and needs a rebase.

**Comment by alicoding** [comment]
rebased

File: components/simple-paypal.jsx [hunk 1]
```diff
@@ -15,14 +15,14 @@ var PaypalInput = React.createClass({
...
     var message = this.props.defaultMessage;
-    if (this.props.value) {
-      message = this.props.message
-                  .replace("{currency_symbol}", this.props.currencySymbol)
-                  .replace("{amount}", this.props.value);
-    }
     return (
       <button type="submit" className="btn large-label-size" id="donate-btn">
-        {message}
+      { this.props.value ?
```
Comment (ScottDowne) [inline components/simple-paypal.jsx]: I bet with a bit of work, this could use the donate button component.

**Pull request merged**